src/main/scala/com/redhat/et/silex/statistics/cramersv.scala

```diff
@@ -0,0 +1,120 @@
+/*
+ * cramersv.scala
+ * author:  RJ Nowling <rnowling@redhat.com>
+ *
+ * Copyright (c) 2016 Red Hat, Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.redhat.et.silex.statistics
+
+import scala.util.Random
+
+/**
+ * "[[https://en.wikipedia.org/wiki/Cram%C3%A9r%27s_V Cramers' V]] is a measure of
+ * association between two nominal variables, giving a value between 0 and +1
+ * (inclusive)." 
+ */
+object CramersV {
```

Comment:
This sort of implicitly assumes there is exactly one function -- `CramersV.apply(...)`, aka `CramersV(...)`.   Are there other related stats that might be associated with this one, if they were ever implemented later?

Other possible designs might be `object CramersVStatistics.cramersV(...)`